src/app/modules/startup/module.nim

```diff
@@ -326,10 +326,11 @@ method emitObtainingPasswordSuccess*[T](self: Module[T], password: string) =
 
 method finishAppLoading*[T](self: Module[T]) =
   self.delegate.finishAppLoading()
+  self.delegate.appReady()
```

Comment:
hmmm, looks like we could use `finishAppLoading`, why have 2 signals?

Reply:
It felt easier, but also it's clearer that we call this only when the app is ready, while `finishAppLoading` is to load the app.